keyboards/nack/keymaps/vial/keymap.c

```diff
@@ -0,0 +1,65 @@
+       _______,     KC_F1,       KC_F2,       KC_F3,       KC_F4,       KC_F5,       KC_F6,       KC_F7,       KC_F8,       KC_F9,       KC_F10,      KC_F11,      KC_F12,
+//  | |---SHIFT----|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|
+       _______,     KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,
+//  | |---CTRL-----|---ALT------|---META-----|---FN_1-----|------------|------------|----FN_2----|--AltGr-----|---( §½ )---|------------|------------|------------|------------|
+       _______,     _______,     _______,     MO(FN),      KC_NO,       KC_NO,       TO(BASE),    _______,     KC_GRV,      KC_NO,       KC_NO,       KC_NO,       KC_NO
+//   \|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|____________|
+    ),
+    [FN] = LAYOUT(
+/*   __________________________________________________________________________________________________________________________________________________________________________
+    | \            \            \            \            \            \            \            \            \            \            \            \            \            \ */
+//  | |-RGB TOGGLE-|-CHANGE RGB-|-RGB HUE UP-|-RGB SAT UP-|------------|------------|------------|------------|------------|------------|------------|------------|-QK_BOOT KBD--|
+       RGB_TOG,     RGB_MOD,     RGB_HUI,     RGB_SAI,     KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,
+//  | |------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|
+       MU_TOG,      KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,
+//  | |------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|------------|-RGB LIGHT +|------------|
+       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       KC_NO,       RGB_VAI,     KC_NO,
+//  | |------------|------------|------------|---FN_1-----|------------|------------|----FN_2----|------------|------------|------------|------------|-RGB LIGHT -|------------|
+       KC_NO,       KC_NO,       KC_NO,       MO(NUM),     KC_NO,       KC_NO,       TO(BASE),    KC_NO,       KC_NO,       KC_NO,       KC_NO,       RGB_VAD,     KC_NO
```

Comment:
https://github.com/vial-kb/vial-qmk/blob/52e5b55a4aec0653df9cd0f5709b0d9b4ea7ce66/docs/feature_layers.md?plain=1#L46
```suggestion
       KC_NO,       KC_NO,       KC_NO,       KC_TRNS,     KC_NO,       KC_NO,       KC_TRNS,    KC_NO,       KC_NO,       KC_NO,       KC_NO,       RGB_VAD,     KC_NO
```
The `TO(BASE)` needs to be transparent because of the `MO(FN)` call on the first layer, and activating a lower layer with `MO(NUM)` from this upper layer isn't going to do anything since you have nothing but `KC_NO` instead of `KC_TRNS`.